Packs/Base/Scripts/DBotMLFetchData/DBotMLFetchData.py

```diff
@@ -1,6 +1,1031 @@
+    else:
+        return_path_adress = ''
+    if last_received_value is not None:
+        received_address_list = re.findall(r'(?<=from )\S+', last_received_value)
+        if len(received_address_list) > 0:
+            received_address = received_address_list[0]
+        else:
+            received_address = ''
+    else:
+        received_address = ''
+    try:
+        from_domain = from_address.split('@')[-1].split('.')[-2] if from_address != '' else ''
+    except Exception:
+        print('error on from_address: {}'.format(from_address))
+        from_domain = ''
+
+    if from_address == '':
+        return_path_same_as_from = float('nan')
+    else:
+        return_path_same_as_from = from_address == return_path_adress
+
+    if from_domain == '' or received_address == '':
+        from_domain_with_received = float('nan')
+    else:
+        from_domain_with_received = from_domain in received_address
+    received_res = {}
+    recevied_headers = [header_dict for header_dict in email_headers if header_dict['name'] == 'Received']
+    count_received = len(recevied_headers)
+    received_res['count_received'] = count_received
+    content_type_value = get_header_value(email_headers, header_name='Content-Type', index=0, ignore_case=True)
+    if content_type_value is not None and ';' in content_type_value:
+        content_type_value = content_type_value.split(';')[0]
+    adress_res = {
+        'return_path_same_as_from': return_path_same_as_from,
+        'from_domain_with_received': from_domain_with_received
+    }
+    res = {}
+    for k, v in res_spf.items():
+        res['spf::{}'.format(k)] = v
+    for k, v in res_dkim.items():
+        res['dkim::{}'.format(k)] = v
+    res['unsubscribe_headers'] = res_unsubscribe
+    for k, v in adress_res.items():
+        res[k] = v  # type: ignore
+    for k, v in received_res.items():
+        res[k] = v
+    res['content-type::{}'.format(content_type_value)] = 1
+    return res
+
+
+def get_attachments_features(email_attachments):
+    res = {}
+    res['number_of_attachments'] = len(email_attachments)
+    all_attachments_names = []
+    for a in email_attachments:
+        attachment_name = a['name']
+        if attachment_name is not None:
+            all_attachments_names.append(attachment_name.lower())
+    all_attachments_names_lengths = [len(name) for name in all_attachments_names]
+    res['min_attachment_name_length'] = min(all_attachments_names_lengths) if len(
+        all_attachments_names_lengths) > 0 else 0
+    res['max_attachment_name_length'] = max(all_attachments_names_lengths) if len(
+        all_attachments_names_lengths) > 0 else 0
+    res['avg_attachment_name_length'] = float(sum(all_attachments_names_lengths)) / len(  # type: ignore
+        all_attachments_names_lengths) if len(all_attachments_names_lengths) > 0 else 0  # type: ignore
+    res['image_extension'] = 0
+    for image_format in ['.jepg', '.gif', '.bmp', '.png']:
```

Comment:
put it in CONST var
missing jpg
fix jpeg
more image extension
bring the extension